src/lib/Annotation.js

```diff
@@ -21,6 +21,13 @@ export default class Annotation {
   /** */
   get resources() {
     if (!this.json || !this.json.resources) return [];
-    return this.json.resources.map(resource => new AnnotationResource(resource));
+    let resourceList;
+    if (Array.isArray(this.json.resources)) {
```

Comment:
Is there a better way to handle this sort of check?

Reply:
We could do something like this:

https://github.com/ProjectMirador/mirador/blob/edd913a3a05a79bbedf0237c9917e6818943cc77/src/lib/AnnotationResource.js#L36

Reply:
Ah, so pull those in from `lodash`?

Reply:
Yep.. we are already using them elsewhere to do similar type things.

Reply:
![:+1:](https://media2.giphy.com/media/ClcWrARkrq1GM/giphy.gif)